dev/com.ibm.ws.kernel.boot.core/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -469,6 +469,14 @@ info.addProductExtension.useraction=No action is required.
 # Too Late for CWWKE0101 - CWWKE0107 they have dual use currently and CWWKE0108 and CWWKE0109 are only used here.
 #
 
+error.dump.rename.fail=CWWKE0200E: The dump artifact {0} could not be renamed to the destination file {1}.
+error.dump.rename.fail.explanation=The dump was still created but the operation has been interrupted.
```

Comment:
Use "was" instead of "has been". Thus, change to:

The dump was created but the operation was interrupted.